test/src/cli/flutter_cli_test.dart

```diff
@@ -209,23 +209,30 @@ void main() {
             ),
             runProcess: process.run,
           ).whenComplete(() {
+            final nestedRelativePath =
+                p.relative(nestedDirectory.path, from: tempDirectory.path);
             verify(() {
               logger.progress(
                 any(
                   that: contains(
                     'Running "flutter packages get" in '
-                    '${nestedDirectory.path}',
+                    '.${p.context.separator}$nestedRelativePath',
                   ),
                 ),
               );
             }).called(1);
 
             verifyNever(() {
+              final ignoredRelativePath = p.relative(
+                ignoredDirectory.path,
+                from: tempDirectory.path,
+              );
+
               logger.progress(
                 any(
                   that: contains(
                     'Running "flutter packages get" in '
-                    '${ignoredDirectory.path}',
+                    './$ignoredRelativePath',
```

Comment:
**Nit**: Use a long string (`'''`) for better readability and understanding of the output.

```dart
'''Running "flutter packages get" in ./$ignoredRelativePath'''
```

_(The same applies to other similar occurrences)_